src/test/resources/schema.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE IF NOT EXISTS student
+(
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(50) NOT NULL,
+    kana_name VARCHAR(50) NOT NULL,
+    nickname VARCHAR(50),
+    email VARCHAR(50) NOT NULL,
+    area VARCHAR(50),
+    age INT,
+    sex VARCHAR(10),
+    remark TEXT,
+    is_Deleted boolean
```

Comment:
大文字小文字の違いはプログラム上大きな差になるので意識しましょう。
```suggestion
    is_deleted boolean
```

Reply:
修正を行いました。また合わせてDBも修正いたしました。
ハッシュコード:[a788223](https://github.com/ireina17/StudentManagement/pull/2/commits/a7882236565c8207996280f3db7d276b9d919f5c)

DB情報
----------------------------------------------
![スクリーンショット 2025-05-21 154350](https://github.com/user-attachments/assets/a8c542a4-8e0d-4fcf-9b37-c13a3f5b9be4)